Makefile

```diff
@@ -47,7 +47,6 @@ install-tools:
 	go install golang.org/x/tools/cmd/goimports@latest
 	# pin for a bug: https://github.com/golangci/golangci-lint/issues/2851
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.2
-	go install -tags 'postgres' github.com/golang-migrate/migrate/v4/cmd/migrate@latest
```

Comment:
Can you replace this with

    go install github.com/jackc/tern@latest

instead? Actually the tern CLI is quite useful and we can use it, just add `tern.conf` into .gitignore and people can configure their database and then issue the tern command to operate on it (migrate, get status, create new migration file).

Reply:
sure :)